evm-contracts/src/v0.6-ovm/FluxAggregator.sol

```diff
@@ -157,7 +158,7 @@ contract FluxAggregator is AggregatorV2V3Interface, Owned {
    * @param _decimals represents the number of decimals to offset the answer by
    * @param _description a short description of what is being reported
    */
-  constructor(
+  function init(
```

Comment:
We covered this on the call, so just shortly here, this is because:

- Saves ~15k deployment gas. Constructor params are appended to bytecode and increase gas cost more than extra deployed bytecode for the `init` function.
- Avoids OVM "safety checker" issues, as it would parse the constructor params as bytecode and could find an unsupported sequence (e.g., SSTORE opcode that is unsupported on OVM)
- Proxy contracts can't construct the underlying implementation contract that was already deployed, it just has the address to work with. But the proxy contract can delegate an `init` call to that impl address.